### ISSUE TITLE
Return format for cmb2_override_meta_value with group fields

### ISSUE THREAD

**Issue by luxdeluks** [WebDevStudios/CMB2#728]
Can anyone help in figuring out how to properly pass values to fields in a group when using a function with cmb2_override_meta_value ?

In other words what should the return be? Passing simple arrays increases the count of groups, aka they repeat, however I can't get the input values to show. 

I've tried many different array combinations with no luck. Should it even be an array?

**Comment by tw2113** [comment]
I'd compare the value passed into the override method and return to match, personally.

**Comment by tw2113** [comment]
Any luck or changes with this one @luxdeluks ?

**Comment by luxdeluks** [comment]
Yes, sorry for not writing anything earlier. It is in fact an array. So with this call

`get_custom_data($data, $id, $arguments, $field)`

we need to grab $arguments. It's basically a multidimensional array with 2 levels with a repeatable group.

The first inner array represents the number of repeats. So if we have

```
array => [0] => array
               [1] => array
               [2] => array
```

Arrays 0, 1, 2 means we have 3 repeats. In order to grab field values for each of those we need to do another lookup inside the inner  array of those. So something like:

$array[0]['field-name']


I'm writing all this from my head so hopefully I didn't forget something. In any case I got it working fine so if anyone stumbles or notices I wrote something wrong feel free to comment off course :)
